eslzArm/managementGroupTemplates/policyAssignments/AUDIT-AppGWWafPolicyAssignment.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
🐪 camelCase the file name name please :)

Reply:
Git doesn't seem to detect case change in filenname?